SetupInstall-Centos7.sh

```diff
@@ -30,6 +46,21 @@ install_software()
 # Check to be root
 need_root
 
+# Raise warning about GPU software installation
+warn_about_gpu_software_requirements
+
+echo ""
+read -r -p "Do you want to proceed with the gpu software installation [y/N] " response
+case "$response" in
+  [yY]) 
+    echo "Installing required gpu software..."
+    install_gpu_software
+  ;;
+  *)
+    echo "Exiting..."
```

Comment:
If I say no, I can still use the whole machinery without GPUs.
There is no need to bail out. And, in fact, you are not bailing out...
Please, echo something like "Continuing without GPU support"